heudiconv/convert.py

```diff
@@ -266,8 +268,8 @@ def convert(items, converter, scaninfo_suffix, custom_callable, with_prov,
                     tmpdir = tempdirs('dcm2niix')
 
                     # run conversion through nipype
-                    res = nipype_convert(item_dicoms, prefix, with_prov, bids,
-                                         tmpdir)
+                    res, prov_file = nipype_convert(item_dicoms, prefix, with_prov,
+                                                    bids, tmpdir)
```

Comment:
is that something new in nipype 1.0 that it would return both?  I am receiving just 
```
-> bids_outfiles = save_converted_files(res, item_dicoms, bids,
(Pdb) p res
<nipype.interfaces.base.InterfaceResult object at 0x7f1b4b5f3350>
```
and this line even test covered, so not quite sure how tests manage(d) to pass etc